src/Hintman/Hint/Position.hs

```diff
@@ -57,19 +59,19 @@ Nothing
 getTargetCommentPosition :: FileDelta -> Int -> Maybe Int
 getTargetCommentPosition FileDelta{..} commentPos = case fileDeltaContent of
     Binary   -> Nothing
-    Hunks hs -> goHunks hs
+    Hunks hs -> goHunks 0 hs
   where
     -- Find the position number i the hunk.
     -- Stops on the first satisfying.
-    goHunks :: [Hunk] -> Maybe Int
-    goHunks [] = Nothing
-    goHunks (Hunk{..}:hs) = case inRange hunkDestRange of
-        Nothing -> goHunks hs
+    goHunks :: Int -> [Hunk] -> Maybe Int
+    goHunks _ [] = Nothing
+    goHunks !acc (Hunk{..}:hs) = case inRange hunkDestRange of
+        Nothing -> goHunks (acc + rangeNumberOfLines hunkDestRange) hs
```

Comment:
Does this includes removed lines? And to follow up, should we include Deleted?

Reply:
No, this doesn't... I realized it only during testing and fixed this. But it still doesn't work properly:

* https://github.com/kowainik/hintman-target/pull/32/files

1. Not that line
2. For some reason, there're 2 comments for this line :thinking: